datasetAPI/get_datasets_test.go

```diff
@@ -63,11 +63,16 @@ func TestSuccessfulGetAListOfDatasets(t *testing.T) {
 					response.Value("items").Array().Element(i).Object().Value("id").Equal(datasetID)
 					checkDatasetResponse(datasetID, response.Value("items").Array().Element(i).Object())
 				}
+
+				if response.Value("items").Array().Element(i).Object().Value("id").String().Raw() == "133" {
+					// user is not authenticated to see this item, if it is returned force failure
```

Comment:
you can do:

```go
t.Log("reason why it failed")
```
to give context to the failure

Reply:
Resolved